packages/react/src/Metafield.tsx

```diff
@@ -190,6 +190,7 @@ export function Metafield<ComponentGeneric extends ElementType>(
  * The `parseMetafield` utility transforms a [Metafield](https://shopify.dev/api/storefront/reference/common-objects/Metafield)
  * into a new object whose `values` have been parsed according to the metafield `type`.
  * If the metafield is `null`, then it returns `null` back.
+ * @deprecated `parseMetafield()` is unsupported and will be removed in a future version. Use `metafieldParser()` instead
  */
 export function parseMetafield(
```

Comment:
should the warn be added when people use this?

Reply:
Ah good point, will add!